storybook/src/components/rich-content/rich-content.content.ts

```diff
@@ -0,0 +1,95 @@
+import { html } from "lit-html";
+
+export function children() {
+  return html`
+    <div class="dso-rich-content">
+      <h1>Kop 1</h1>
+      <h2>Kop 2</h2>
+      <h3>Kop 3</h3>
+      <h4>Kop 4</h4>
+      <p>We ondersteunen <em>emphasis</em>, <strong>strong</strong>, <i>italic</i>, <b>bold</b> en <u>underline</u>.</p>
+      <p>
+        Verder doen we aan <sub>lage</sub> en <sup>hoge</sup> letters en voegen we wel eens wat <del>weg</del
+        ><ins>toe</ins>.
+      </p>
+      <p>Een {% render '@anchor', {label: 'externe link', url: '#', modifier: 'extern') %} maak je zo</p>
```

Comment:
Hier willen we de anchor renderen met de anchorTemplate. Ook andere handlebars dienen gewijzigd te worden naar templates.